toolchains/regenerate.sh

```diff
@@ -38,7 +38,7 @@ esac
 
 # Bazel query is the right command so bazel won't fail itself.
 # Keep bazel versions here at most two: current master version, next version
-for BAZEL_VERSION in "3.6.0" "3.7.0"; do
+for BAZEL_VERSION in "3.7.2"; do
```

Comment:
At time of applying this patch, the transition will be from 3.6.0+3.7.2 -> only 3.7.2

because on windows, the build will fail if we attempt to regenerate 3.6.0 after upgrading llvm to 11.0.0 on windows.

Reply:
sgtm